experiments/RAFT-diffusion/README.md

```diff
@@ -0,0 +1,105 @@
+# RAFT-Diffusion
+
+
+In this folder, we provide an example to show that how does RAFT work on diffusion models. We will also include these script into LMFlow framework into the LMFlow APIs in the future.
+
+
+The requirements are shown below.
```

Comment:
Can we provide a `requirements.txt` in this path?

Reply:
Done